adserver/models.py

```diff
@@ -1599,6 +1612,8 @@ class Offer(AdBase):
 
     """Contains data on ad views."""
 
+    MAX_VIEW_TIME = 5 * 60  # seconds
```

Comment:
The client also supports this, but we're validating on the server as well in case of misbehaving clients?

Reply:
This is correct. The client could send anything.